HHS/OPRE-OPS
# feat: ✨ Approver Change Notification icon with CR total

## What changed

- adds Approver Change Request icon with total of Change Requests to review
- adds unit-tests
- updates e2e test to look for them

## Issue

#1471 

## How to test

1. Add a change requests to Agreement 1/9
2. Go to Review Tab
3. See red circle with the number of change requests
4. Approve/Decline the change request(s)
5. Review Tab should not show red circle when all CR have been handled

## Screenshots

https://github.com/user-attachments/assets/9b4820d8-439d-4814-9718-818133281c84

## Definition of Done Checklist
